pages/library_implementation.md

```diff
@@ -0,0 +1,125 @@
+We have various types of resources that either form a collection or belong to common groups of resources. These include the African Storybooks, Comic Books, CK-12 Textbooks, TED-Ed videos, Khan Academy Videos, etc. If you are uploading one of these resources, please read the specific guides we have created and follow the instructions. This maintains uniformity in resource details and makes searches easier for Learners.
+
+You may need to use a format converter to put your resource in the correct format before uploading. There are numerous free online converters which you may use.
+
+**IMPORTANT:** Until all Nations can communicate with each other and resources can easily be shared between Nations, resources that are appropriate for uploading to more than one Nation (i.e. Somaliabell, and Kenyabell) must be uploaded into each Nation using the same cataloging information each time. This can be a time consuming process, but it is currently the only way we can ensure that resources are available in each Nation.
+
+## How do I name a file so that it can be upoaded?
+
+The following file naming conventions should be used for any new file uploaded to PLANET:
+
+* Rename the file with a meaningful and unique file name that can be easily associated with the title/content of the resource. For example: Alcott_LittleWomen.pdf
+* Use alpha-numeric characters only. Do not use special characters (except for "-" and "_").
+* Do not include spaces in the file name. Instead of a space, use an underscore.
+* Do not use characters or letters that do not exist in the English language. For example, avoid naming your files in Arabic without transliteration.
+* If there is a series of resources, the first part of the file name should be consistent (e.g., module title), followed by the number in the series, followed by the title of the specific file (e.g., ComputerBasics01_GettingToKnowComputers.pdf, ComputerBasics02_UnderstandingOperationSystems.pdf, etc.).
+* Use leading zeros (i.e., 01, 02, etc. or 001, 002, etc.) for documents that are part of a series so that they are sorted numerically.
+
+## Are there resources that have specific rules for uploading?
+
+Yes. These resources are covered in an additional manual titled, XXXXX. Please read that manual to address specific questions on uploading in other languages, in additional formats, or in existing collections with special permission.
+
+## How do I create and use collections?
+
+Collections can be used by Nation administrators and managers to categorize materials by topic. Collections make it easy to group resourcers so that they are easily found by Learners. **NOTE:** Collections cannot be transferred between Nations and Communities so any changes to a collection are unique to that specific instance of PLANET. Collections can only be created or changed by administators.
+
+1. Log in to the Manager account.
+2. Click “Library” on the top navigation bar and then “Collections”.
+
+![](uploads/images/Collection.png)
```

Comment:
I believe you need to indent the link to image to make the ordered list working correctly on MDwiki.